mlir/test/Dialect/SparseTensor/sparse_vector_mv.mlir

```diff
@@ -1,3 +1,5 @@
+// UNSUPPORTED: target={{.*}}
```

Comment:
@aartbik I think it is caused by the newly added `CanonicalizationPass`, which might change the code a little bit.